### PR TITLE
Add Crust Rocky Network Page

### PR DESCRIPTION
**Pull request by MyronFanQiu**

### CODE REVIEW

File: docs/build-crust-sdk.md [hunk 1]
```diff
@@ -81,22 +81,7 @@ The PolkadotJS API document can be found [here](https://polkadot.js.org/docs/api
...
-  
-- Extrinsics: contains extrinsics execution API list for different modules.
-  - `market`: contains storage market related API, such as p`laceStorageOrder`, `calculateReward`, `addPrepaid`, etc.
-  - `staking`: contains GPoS related API, such as `bond`, `unbond`, `validate`, `gurantee`, etc.
-  - `swork`: contains node storage proof related API, such as `createGroup`, `joinGroup`, etc.
-  - `session`, `balances`, `utility`, etc: contains typical blockchain related API, such as session.`setKeys`, balances.`transfer`, utility.`batch`, etc.
-
-![api-placeorder](assets/build/api-placestorageorder.png)
+Please visit [Crust Docs](http://101.132.117.183:8080/) to check all Crust API. You can check the [`Constants`](http://101.132.117.183:8080/substrate/constants) to get all constants stored on chain. You can learn how the chain state organazied on [`Storage`](http://101.132.117.183:8080/substrate/storage) page and all supported extrinsices on [`Extrinsics`](http://101.132.117.183:8080/substrate/extrinsics). You can check the [`Events`](http://101.132.117.183:8080/substrate/events) and [`Errors`](http://101.132.117.183:8080/substrate/errors) as well if neccessary.
```
Comment (MyronFanQiu) [inline docs/build-crust-sdk.md:84]: change this ip address

File: docs/build-rocky-guidance.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+title: Crust Rocky Network
+sidebar_label: Crust Rocky Network
+---
+
+Welcome to Crust Rocky Network. Crust Rocky Network is a test network for developers to test their application easily and quickly.
+
+Here you will learn how to join the Crust Rocky Network, get the Rocky CRU from the faucet and important parameters of Crust Rocky Network.
+
+## How to play with the Crust Rocky Network.
+You can join and check the Crust Rocky Network from our [apps](https://apps.crust.network/?rpc=wss%3A%2F%2Frocky-api.crust.network#/accounts). It's exact same with our Crust Maxwell Preview Network. You can create your account first and copy the account address.
```
Comment (MyronFanQiu) [inline docs/build-rocky-guidance.md:12]: change this link

File: docs/build-rocky-guidance.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    ports:
+      - "30333:30333"
+      - "9933:9933"
+      - "9944:9944"
+```
+
+If you want to build and run a chain from source code, please check the [readme](https://github.com/crustio/crust/blob/master/README.md) in the [crust repo](https://github.com/crustio/crust).
+
+## API Docs
+You can check [Crust Docs](http://101.132.117.183:8080/) website to learn supported api for our chain.
```
Comment (MyronFanQiu) [inline docs/build-rocky-guidance.md:88]: change this ip address

File: website/translated_docs/zh-CN/build-crust-sdk.md [hunk 4]
```diff
@@ -81,22 +81,7 @@ The PolkadotJS API document can be found [here](https://polkadot.js.org/docs/api
...
-  
-- Extrinsics: contains extrinsics execution API list for different modules.
-  - `market`: contains storage market related API, such as p`laceStorageOrder`, `calculateReward`, `addPrepaid`, etc.
-  - `staking`: contains GPoS related API, such as `bond`, `unbond`, `validate`, `gurantee`, etc.
-  - `swork`: contains node storage proof related API, such as `createGroup`, `joinGroup`, etc.
-  - `session`, `balances`, `utility`, etc: contains typical blockchain related API, such as session.`setKeys`, balances.`transfer`, utility.`batch`, etc.
-
-![api-placeorder](assets/build/api-placestorageorder.png)
+Please visit [Crust Docs](http://101.132.117.183:8080/) to check all Crust API. You can check the [`Constants`](http://101.132.117.183:8080/substrate/constants) to get all constants stored on chain. You can learn how the chain state organazied on [`Storage`](http://101.132.117.183:8080/substrate/storage) page and all supported extrinsices on [`Extrinsics`](http://101.132.117.183:8080/substrate/extrinsics). You can check the [`Events`](http://101.132.117.183:8080/substrate/events) and [`Errors`](http://101.132.117.183:8080/substrate/errors) as well if neccessary.
```
Comment (MyronFanQiu) [inline website/translated_docs/zh-CN/build-crust-sdk.md:84]: change this ip address

File: website/translated_docs/zh-CN/build-rocky-guidance.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+title: Crust Rocky Network
+sidebar_label: Crust Rocky Network
+---
+
+Welcome to Crust Rocky Network. Crust Rocky Network is a test network for developers to test their application easily and quickly.
+
+Here you will learn how to join the Crust Rocky Network, get the Rocky CRU from the faucet and important parameters of Crust Rocky Network.
+
+## How to play with the Crust Rocky Network.
+You can join and check the Crust Rocky Network from our [apps](https://apps.crust.network/?rpc=wss%3A%2F%2Frocky-api.crust.network#/accounts). It's exact same with our Crust Maxwell Preview Network. You can create your account first and copy the account address.
```
Comment (MyronFanQiu) [inline website/translated_docs/zh-CN/build-rocky-guidance.md:12]: change this link

File: website/translated_docs/zh-CN/build-rocky-guidance.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    ports:
+      - "30333:30333"
+      - "9933:9933"
+      - "9944:9944"
+```
+
+If you want to build and run a chain from source code, please check the [readme](https://github.com/crustio/crust/blob/master/README.md) in the [crust repo](https://github.com/crustio/crust).
+
+## API Docs
+You can check [Crust Docs](http://101.132.117.183:8080/) website to learn supported api for our chain.
```
Comment (MyronFanQiu) [inline website/translated_docs/zh-CN/build-rocky-guidance.md:88]: change this ip address

File: docs/build-crust-sdk.md (hunk 1 above)
Comment (zikunfan) [inline docs/build-crust-sdk.md:84]: After a settled domain

**Review by zikunfan — CHANGES_REQUESTED** [review]

**Review by zikunfan — APPROVED** [review]

**Review by zikunfan — APPROVED** [review]

**Pull request merged**